packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2511,6 +2511,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     if (_showCaretOnScreenScheduled) {
       return;
     }
+    // if cursor is inactive, e.g. while selecting text, do not jump away
+    if (!_cursorActive) {
```

Comment:
Should we move this to line 2521? We still schedule the "scroll to reveal cursor" task but if there's no cursor by then (at the end of the frame) then we don't actually scroll the text field.

Reply:
I though about not initiating the scheduler, but regarding to result, it would still the same :) I think we could change